### PR TITLE
vrepl: fix array method call errors (fix #15769)

### LINKED ISSUE #15769: vrepl bug in array methods pop and delete_last
```v
>>> mut arr := [1, 2, 3]
>>> a.pop()
>>> a
[1, 2, 3]
>>> a.delete_last()
>>> a
[1, 2, 3]
```

### PR DESCRIPTION
**Pull request by yuyi98**
This PR fix array method call error in vrepl (fix #15769).

- Fix array method call error.
- Add test.

```v
PS D:\vlang\v> v
 ____    ____
 \   \  /   /  |  Welcome to the V REPL (for help with V itself, type  exit , then run  v help ).
  \   \/   /   |  Note: the REPL is highly experimental. For best V experience, use a text editor,
   \      /    |  save your code in a  main.v  file and execute:  v run main.v
    \    /     |  V 0.3.1 7e69619 . Use  list  to see the accumulated program so far.
     \__/      |  Use Ctrl-C or  exit  to exit, or  help  to see other available commands.

>>> mut arr := [1, 2, 3]
>>> arr
[1, 2, 3]
>>> arr.delete_last()
>>> arr
[1, 2]
>>> arr.clear()
>>> arr
[]
>>>
```

### CODE REVIEW

File: vlib/v/tests/repl/error_exitasdfasdf.repl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 exitasdfasdf
 ===output===
-error: undefined ident: `exitasdfasdf`
+.vrepl.v:7:9: error: undefined ident: `exitasdfasdf`
```
Comment (spytheman) [inline vlib/v/tests/repl/error_exitasdfasdf.repl:3]: hm, weird ... the purpose of `convert_output` is to strip that prefix, but it did not work for some reason 🤔

Comment (yuyi98) [reply vlib/v/tests/repl/error_exitasdfasdf.repl:3]:
```v
		} else if line.contains('.vrepl.v:') {
			// Ensure that .vrepl.v: is at the start, ignore the path
			// This is needed to have stable .repl tests.
			idx := line.index('.vrepl.v:') or { panic(err) }
			content += endline_if_missed(line[idx..])
```

Comment (yuyi98) [reply vlib/v/tests/repl/error_exitasdfasdf.repl:3]: Whether to filter out before and after messages and display only error messages?

Comment (yuyi98) [reply vlib/v/tests/repl/error_exitasdfasdf.repl:3]: I found the reason, I'll change it.

**Review by spytheman — COMMENTED** [review]
Very clever workaround, but try the following:
```
println('hello')
fn abc(x int) { println(x) }
abc(123)
abc(456)
println('hello')
```

**Comment by yuyi98** [comment]
There is still something wrong and I will try to fix it.

**Comment by yuyi98** [comment]
@spytheman  Thanks !!!  At present, there is an error in the VREPL test on the Windows platform and it cannot be used normally. Could you help to solve the problem?

**Comment by spytheman** [comment]
hm, the CI did not find the problem 🤔
Should I revert the merge?

**Comment by yuyi98** [comment]
I'll push another PR to fix it.

**Comment by spytheman** [comment]
> I'll push another PR to fix it.

ok

**Comment by yuyi98** [comment]
I meas I cannot test VREPL normally on windows now.

**Pull request merged**